src/kernels/composable_kernel/include/tensor_operation/blockwise_generic_tensor_slice_copy.hpp

```diff
@@ -59,8 +59,8 @@ struct BlockwiseGenericTensorSliceCopy_v4
         static_assert(BlockSize >= mThreadClusterDesc.GetElementSize(),
                       "wrong! BlockSize too small");
 
-        if(BlockSize == mThreadClusterDesc.GetElementSize() or
-           get_thread_local_1d_id() < mThreadClusterDesc.GetElementSize())
+        // if(BlockSize == mThreadClusterDesc.GetElementSize() or
```

Comment:
@asroy How we deal with this hijack? in a dedicated PR?

Reply:
Is this still causing branching?

Reply:
No. I reverted changes already.